tests/h2-tests/tests/push_promise.rs

```diff
@@ -203,7 +316,6 @@ fn recv_push_promise_with_wrong_authority_is_stream_error() {
 #[test]
 fn recv_push_promise_skipped_stream_id() {
```

Comment:
Would you be able to implement this test?

The idea is that the server sends a push promise with a stream ID of 8 or something. This works.

Then the server attempts to send a push promise of 4, which should not work because skipped IDs are implicitly closed.

Reply:
Aren't these tests already implemented? Or am I missing something

Reply:
Hmm... yeah, I'm not sure what I was thinking.